api/v1alpha1/gatewayclass_types.go

```diff
@@ -90,6 +94,10 @@ type GatewayClassSpec struct {
 	// intentionally a pointer to support the nil behavior (only local Routes
 	// allowed).
 	//
+	// When a Route is selected by a Gateway in a namespace that is forbidden by
```

Comment:
suggest "from a namespace that is not allowed by this selector"

Reply:
Updated, thanks!